server/docker-compose.yml

```diff
@@ -10,8 +10,13 @@ api:
     - VAULT_IV=kfsdfopweu09fewjlhfksdjifjpoweop
     - ACME_ENDPOINT=https://acme-staging.api.letsencrypt.org/
     - INITIAL_ADMIN_CODE=initialadmincode
+    - DEBUG=1
```

Comment:
It's handy to see debug logs during development. And I figure no-one uses these compose files in any other env than dev purposes.